docs/src/transfer-hook-interface/introduction.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Introduction
```

Comment:
nit to make it clear what the page is:
```suggestion
title: Transfer Hook Interface Introduction
```

Reply:
Thinking of dropping `Introduction` altogether from this title.